ruby/basics/conditional_logic.md

```diff
@@ -1,43 +1,246 @@
 # Conditional Logic
 A brief summary about what this lesson is about and why the topics or concepts it covers are important.
 
+This lesson is all about controlling the flow of your code. The concept is pretty simple. You have some code that you only want to execute under specified conditions, so you need a way for the computer to check whether those conditions have been met. Conditional logic can be found everywhere in everyday life. Ever had to tidy your room before being allowed to play video games? That's your mother setting up a nice conditional statement that might look like this in a computer program...
+
+```ruby
+if room_tidy == true
+  ask mother if I can play video games
```

Comment:
I think the if within the condition might confuse people. We could change it to this to avoid that:

```
if room_tidy == true
 "I can play video games"
```